handlers_global.go

```diff
@@ -72,7 +72,7 @@ func handleTraceRequest(ctx context.Context, stats *statsd.Client, w http.Respon
 	span, err = tracer.ExtractRequestChild("/spans", r, "veneur.opentracing.spans")
 	if err != nil {
 		log.WithError(err).Info("Could not extract span from request")
-		span = tracer.StartSpan("/spans", trace.NameTag("veneur.opentracing.spans")).(*trace.Span)
+		span = tracer.StartSpan("veneur.opentracing.spans").(*trace.Span)
```

Comment:
So in this case, we're not going to have the `/spans` endpoint actually appear in the name? That seems backwards, no?